xml/defaultConfig.xml

```diff
@@ -112,4 +112,23 @@
         <!-- <treeTransportMission></treeTransportMission> -->
         <!-- <destructibleRockMission></destructibleRockMission> -->
     </customRewards>
+
+    <customMaxType>
+        <!-- <baleMission></baleMission> -->
+        <!-- <baleWrapMission></baleWrapMission> -->
+        <!-- <plowMission></plowMission> -->
+        <!-- <cultivateMission></cultivateMission>  -->
+        <!-- <sowMission></sowMission> -->
+        <harvestMission>12</harvestMission>
+        <!-- <hoeMission></hoeMission> -->
+        <!-- <weedMission></weedMission> -->
+        <!-- <herbicideMission></herbicideMission> -->
+        <!-- <fertilizeMission></fertilizeMission> -->
+        <!-- <mowMission></mowMission> -->
+        <!-- <tedderMission></tedderMission> -->
+        <!-- <stonePickMission></stonePickMission> -->
+        <deadwoodMission>1</deadwoodMission>
```

Comment:
would `<deadwoodMission>0</deadwoodMission>` also work for a player to remove those type of missions from getting generated?